packages/generators/app/src/create-project.ts

```diff
@@ -136,12 +144,10 @@ export default async function createProject(
 
   try {
     if (scope.installDependencies !== false) {
-      const runner = runInstall(scope);
+      const runner = await runInstall(scope);
 
-      runner.stdout?.on('data', logInstall);
-      runner.stderr?.on('data', logInstall);
-
-      await runner;
+      logInstall(runner.stdout);
+      logInstall(runner.stderr);
```

Comment:
@innerdvations This is probably wrong, but I would want to catch the error in `runInstall` and need to await therefore in there. Happy to change that.